oshdb-api/src/main/java/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapAggregator.java

```diff
@@ -759,7 +762,8 @@ public X setValue(X value) {
   @Contract(pure = true)
   public <R> MapAggregator<U, R> map(SerializableFunction<X, R> mapper) {
     return this.copyTransform(this.mapReducer.map(inData -> {
-      //noinspection unchecked – trick/hack to replace mapped values without copying pair objects
+      @SuppressWarnings("unchecked")
+      // trick/hack to replace mapped values without copying pair objects
```

Comment:
Usually I prefer to have the @SuppressWarnings directly following the affected line. 
So in this sense the comment would come before or in the same line as  @SuppressWarings:
```
// trick/hack to replace mapped values without copying pair objects
@SuppressWarnings("unchecked")
IndexValuePair<U,R> outData = (IndexValuePair<U,R>)inData;
```
But this is only my personal preference and does not hinder the approval!

Reply:
:thinking:  hmm, I did it like this because it is otherwise not clear that the comment is related to the `@SuppressWarnings` statement: When the comment is in the line above, it could also be just a regular inline comment. Short comments about the suppress statement you would also put _after_ the annotation (`@SuppressWarnings("unchecked") // bla bla`), wouldn't you?

But I guess that's ultimately a matter of taste. :shrug: